book/src/concepts/inheritance.md

```diff
@@ -15,6 +15,7 @@ To support creating such subclasses directly from within Rust, CXX-Qt provides y
 To access the methods of a base class in Rust, use the `#[inherit]` macro.
 It can be placed in front of a function in a `extern "RustQt"` block in a `#[cxx_qt::bridge]`.
 
+TODO: all of these examples need sorting out
```

Comment:
book needs checking

Reply:
tidied it up, but probably makes sense to rewrite the book similar to CXX with `extern "C++Qt"` and `extern "RustQt"` topics etc.

Reply:
Split that into #582